model/gen-ai/registry.yaml

```diff
@@ -253,6 +255,91 @@ groups:
             Client-side operations are actions taken on the user's end or within the client application.
           Datastore: A tool used by the agent to access and query structured or unstructured external data for retrieval-augmented tasks or knowledge updates.
         examples: ['function', 'extension', 'datastore']
+
+      - id: gen_ai.tool.call.arguments
+        stability: development
+        type: any
+        brief: Parameters passed to the tool call.
+        note: |
+          > [!WARNING]
+          > This attribute may contain sensitive information.
+
+          It's expected to be an object - in case a serialized string is available
```

Comment:
Is it allowed to be a list or scalar like int/string?

Reply:
@aabmass : Individual arguments can certainly use simple data types such as int or string. During the initial SIG discussions, it was agreed that we don’t need to explicitly enumerate all possible data types. Instead, providing a few representative examples would be sufficient to illustrate the supported data formats for users.

Reply:
That sounds good to me, but the text says "It's expected to be an object"

Reply:
yes, its complex object returned as JSON with arguments as key : value pair. Each argument can be a simple data-type of int/string etc.